pkg/hash/v1alpha1/hash.go

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
IMO for helper functions ...we don't need versioning. We can have these helpers inside `pkg/util`.

Reply:
@prateekpandey14 It has been decided to avoid the terms like `util` & `common`.